lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py

```diff
@@ -339,10 +339,10 @@
                              q3dpJxJga76h9BZkK2BJJrDzSO6DDn6yKads017OZBZ1YZWyq1cW4iuvA=="
         tags:
             description:
-                - Tags
+                - Tags assigned to the resource. Dictionary of string:string pairs.
             returned: always
-            type: complex
-            sample: {}
+            type: dict
+            sample: { "tag1": "abc" }
```

Comment:
I can't tell if there should be a space or not. If there should be a space, then add one in line 342 above (`"string": "string"`). If there should NOT be a space, then remove it here (`"tag1":"abc"`).